src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,9 +17,22 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.0.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
     <PackageReference Include="System.Text.Json" Version="4.6.0" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

Comment:
I'm not sure multi-targeting is needed because of these new references. They are all still maintained as netstandard.

Reply:
You're right. Not sure why I had to introduce multi targeting during my testing, but now I see that its not really needed. We have two options:
1. Upgrade `Microsoft.Extensions.DependencyInjection.Abstractions` to 5.x so that we can use 5.x DataProtection packages.
2. Add 2.x DataProtection packages if we don't want to upgrade `Microsoft.Extensions.DependencyInjection.Abstractions`.

Reply:
Updated `Microsoft.Extensions.DependencyInjection.Abstractions` to 3.x and added 3.x DataProtection packages.